OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -291,11 +292,79 @@ public override bool HandleKeyPress(KeyInput e)
 					return ToggleMute();
 				else if (key == Game.Settings.Keys.TogglePlayerStanceColorsKey)
 					return TogglePlayerStanceColors();
+				else if (key == Game.Settings.Keys.NextTrack)
```

Comment:
Okay I get it. 
I started using `WorldInteractionControllerWidget ` because it had `HandleKeyPress(KeyInput e)` and MusicPlayerLogic don't and couldn't find any music player widget with handle method. So where should I handle the keypress?